crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -483,21 +483,27 @@ impl Room {
     }
 
     /// Get the read receipt as a `EventId` and `Receipt` tuple for the given
-    /// `user_id` in this room.
+    /// `thread` and `user_id` in this room.
     pub async fn user_read_receipt(
```

Comment:
Since the two methods changed in this file are public API, I wonder what you think about this change, @poljar. If we wanted to be more conservative, we could have separate functions for unthreaded and threaded receipts. But maybe it's actually cleaner this way..

Reply:
I agree that it's cleaner this way. If we have separate methods users might not notice that there are two and use the wrong one.

This way we force people to think about the context of the read receipt.